api/src/pcapi/routes/pro/collective_offers.py

```diff
@@ -36,18 +37,32 @@
 @spectree_serialize(
     response_model=collective_offers_serialize.ListCollectiveOffersResponseModel,
     api=blueprint.pro_private_schema,
+    query_params_as_list=["status"],
 )
 def get_collective_offers(
     query: collective_offers_serialize.ListCollectiveOffersQueryModel,
 ) -> collective_offers_serialize.ListCollectiveOffersResponseModel:
+    if query.status:
+        assert isinstance(query.status, list)
+        # Only 1 status is permitted. Multiple status tu be implemented very soon
```

Comment:
to. ou will be implemented.
Le mettre dans un `#TODO:` serait peut être plus facile.